docker/docker-compose.yml

```diff
@@ -10,6 +10,8 @@ services:
       - RPC_PORT=9933
       - WS_PORT=9943
       - PORT=30333
+      - VALIDATOR_PORT=30342
+      - PUBLIC_VALIDATOR_ADDRESS=127.0.0.1:30342
```

Comment:
Why not start the numbering from `30343` which looks like the default otherwise?

Reply:
Started out as a typo, and then I rolled with it. You are right though, this is just confusing, fixed.